tools/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.unicode.cldr</groupId>
+  <artifactId>cldr-all</artifactId>
+  <version>38.0-SNAPSHOT</version>
+  <name>CLDR Top Directory</name>
+  <packaging>pom</packaging>
+  
+  <modules>
+    <module>java</module>
+    <module>cldr-apps</module>
+  </modules>
```

Comment:
Since this is the parent (root?) pom.xml, this is where you can "refactor" (de-duplicate) specifying the version numbers of commonly-used dependencies and plugins via [`dependencyManagement`](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management).

Similarly, using `<dependencyManagement>` in `tools/cldr-apps/pom.xml` probably isn't doing much for you, as it is a child/leaf pom.xml.

Also, if you refactor `./java/pom.xml` and `./cldr-apps/pom.xml` to de-duplicate using the parent-child inheritance, then you can run one of the "child" pom.xmls (ex: `./java/pom.xml`) with the proper inerhitance/interpolation in effect by running this command in the directory of the root pom.xml:  `mvn -am -pl java/pom.xml`.

I think the only catch with Maven is that the version number of artifact that the pom.xml defines can't be inherited, and it must be hard-coded.

I don't know if this is useful, but I know that there are [Maven plugins](http://maven.apache.org/maven-release/maven-release-plugin/index.html) out there to manage the incrementing of versions from N to (N+1)-SNAPSHOT to N+1, etc.  They can can manage the version value in the submodule pom.xml's as well.

Reply:
> Similarly, using <dependencyManagement> in tools/cldr-apps/pom.xml probably isn't doing much for you, as it is a child/leaf pom.xml.

This turned out to help the build be able to find it as a sibling dependency.

Will do the rest, I quickly scrambled my workspace trying to do this refactoring. 

I'd really like to rename 'java' to something, perhaps 'cldr4j' or 'cldr-tools'. Otherwise it's misnamed.

Reply:
corrected… push coming